config/foreman-proxy-content.migrations/240531091300_foreman_proxy_tftp_host_config.rb

```diff
@@ -0,0 +1,10 @@
+if answers['foreman_proxy']
```

Comment:
I always like to think about the data types and this is a `Variant[Boolean, Hash]`. In other words, this can be `true` as well and cause the later code to crash. `true` means use all defaults and you don't need a migration. That's why most migrations use `.is_a?(Hash)`
```suggestion
if answers['foreman_proxy'].is_a?(Hash)
```

Reply:
Done